lib/routes/isct/namespace.ts

```diff
@@ -4,11 +4,11 @@ export const namespace: Namespace = {
     name: 'Institute of Science Tokyo',
     url: 'isct.ac.jp',
     lang: 'ja',
-    description: `
-:::tip
-    支持通过category参数筛选新闻类别。详情请查看[指南](https://docs.rsshub.app/zh/guide/parameters#%E5%86%85%E5%AE%B9%E8%BF%87%E6%BB%A4)。
-    You can filter news by category through the category parameter. For more information, please refer to the [guide](https://docs.rsshub.app/guide/parameters#filtering).
-:::`,
+    description: `:::tip
+支持通过 category 参数筛选新闻类别。详情请查看 [指南](https://docs.rsshub.app/zh/guide/parameters#%E5%86%85%E5%AE%B9%E8%BF%87%E6%BB%A4) 。
+    
+You can filter news by category through the category parameter. For more information, please refer to the [guide](https://docs.rsshub.app/guide/parameters#filtering).
+    :::`,
```

Comment:
```suggestion
:::`,
```
Indent the markdown content with either 0 or 2 spaces otherwise they won't be rendered properly.